client/src/ui/molecules/search/index.scss

```diff
@@ -4,6 +4,15 @@
 .header-search {
   max-width: 100%;
 
+  .search-results {
+    max-height: calc(100vh - var(--top-nav-height) - 7rem);
```

Comment:
Maybe it makes sense to add a short comment to explain the `7rem`? (I guess it is about the Theme switcher + some space at the bottom?!)

Unfortunately though, it doesn't seem to be enough for unauthenticated users:

<img width="758" alt="image" src="https://user-images.githubusercontent.com/495429/202709605-505c694d-500c-42f7-96e4-26f1d38415f7.png">